### PR TITLE
Psalm sa

### PR DESCRIPTION
**Pull request by greg0ire**
I'm blocked on that one, creating a PR anyway to demonstrate a bug

### CODE REVIEW

**Comment by greg0ire** [comment]
There was no bug, my issue is fixed :)

File: src/SqlFormatter.php [hunk 1]
```diff
@@ -1045,7 +1045,7 @@ private static function getNextToken(string $string, ?array $previous = null) :
         ];
     }
 
-    private static function getQuotedString(string $string) : string
+    private static function getQuotedString(string $string) : ?string
```
Comment (goetas) [inline src/SqlFormatter.php]: I'm curious if this is hiding a long-standing bug, and `$ret` should be initialized as `$string`, so the return type will stay `string` (not nullable)

Comment (greg0ire) [reply src/SqlFormatter.php]: Just from reading the signature it seems you're right, I will look into it!

Comment (greg0ire) [reply src/SqlFormatter.php]: Do you like it better like this?

**Pull request merged**